src/lib/attribution_classes.js

```diff
@@ -0,0 +1,66 @@
+export default {
+  unique: {
+    id: "unique",
+    name: "Unique",
+    info: "One of a kind piece.",
+    short_description: "This is a unique work.",
+    long_description: "One of a kind piece, created by the artist.",
+  },
+  "limited edition": {
+    id: "limited_edition",
+    name: "Limited edition",
+    info: "Original works created in multiple.",
+    short_description: "This is part of a limited edition set.",
+    long_description: [
+      "Original works created in multiple with direct involvement of the artist.",
+      "Generally, less than 150 pieces total.",
+    ].join(" "),
+  },
+  "made-to-order": {
+    id: "made-to-order",
+    name: "Made-to-order",
+    info: "A made-to-order piece.",
+    short_description: "This is a made-to-order piece.",
+    long_description: "A piece that is made-to-order, taking into account the collector’s preferences.",
+  },
+  reproduction: {
+    id: "reproduction",
+    name: "Reproduction",
+    info: "Reproduction authorized by artist’s studio or estate.",
+    short_description: "This work is a reproduction.",
+    long_description: [
+      "Reproduction of an original work authorized by artist’s studio or estate.",
+      "The artist was not directly involved in production.",
+    ].join(" "),
+  },
+  "editioned multiple": {
+    id: "editioned multiple",
```

Comment:
This id has a space but the one on [L10](https://github.com/artsy/metaphysics/pull/955/files#diff-bd6d53f9c5aff2f8fd7f40cf98028d60R10) has an underscore. Intentional?

Do these need to have parity [with these](https://github.com/artsy/gravity/blob/master/app/models/domain/attribution.rb#L8-L14)?

(Same observation for some other id's in this file)

Reply:
That was not intentional. Technically we do not even need this Id here, but I though it might be nice to get is as part of the object as something to refer back to in gravity if we need to. Corrected.